### ISSUE TITLE
summary.lmList issues

### ISSUE THREAD

**Issue by bbolker** [lme4/lme4#377]
`summary.lmList` can fail when some groups have singular fits.  This is really an upstream problem in `nlme`, since we copy the summary methods for our own use ...

```
p2 <- subset(Pixel, Dog %in% c(7,8,9))
p3 <- subset(Pixel, Dog %in% c(7,8))  ## without problematic case
library("nlme")
L1 <- lmList(pixel~day+I(day^2)|Dog, p2)
try(summary(L1))
L2 <- lme4::lmList(pixel~day+I(day^2)|Dog, p2)
try(summary(L2))
```

**Comment by bbolker** [comment]
It's a built-in data set from the `nlme` package. `?nlme::Pixel`  (and/or `data("Pixel", package = "nlme")`)

>  The ‘Pixel’ data frame has 102 rows and 4 columns of data on the pixel intensities of CT scans of dogs over time

**Comment by bbolker** [comment]
Can we close this now?

**Comment by annahuynhly** [comment]
No; what I solved didn't deal with the issue but some naming issue that I noticed alongside trying to solve this issue. There is still a problem here, that copilot didn't fix (maybe Claude can? I'll put it on my list...)

**Comment by annahuynhly** [comment]
Close; it was solved by GH #970, and for future reference: 'singular fits' is not an appropriate name; should've said "for rank deficient models..."
